dask_geopandas/core.py

```diff
@@ -1,11 +1,28 @@
 import numpy as np
+import pandas as pd
 
 import dask.dataframe as dd
 from dask.dataframe.core import _emulate, map_partitions, elemwise
 from dask.utils import M, OperatorMethodMixin, derived_from, ignore_warning
 
 import geopandas
-from shapely.geometry.collection import GeometryCollection
+from shapely.geometry.base import BaseGeometry
+
+
+def _set_crs(df, crs):
+    """Return a new object with crs set to ``crs``"""
+    df = df.copy(deep=False)
+    df.crs = crs
+    return df
+
+
+def _finalize(results):
+    if isinstance(results[0], (geopandas.GeoSeries, geopandas.GeoDataFrame)):
+        output = pd.concat(results)
+        output.crs = results[0].crs
+        return output
```

Comment:
Ah, this might actually be a geopandas bug then that concatting GeoSeries does not preserve the CRS (will look into that)

Reply:
It indeed is -> https://github.com/geopandas/geopandas/issues/1340

Reply:
Oh cool. I included the fix in this PR anyways, but we can take it out later.